### PR TITLE
[UG] Update Find Command

### PR DESCRIPTION
**Pull request by lyeyixian**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -58,6 +63,7 @@ Command | Parameters | Description
 `delete -OBJECT` | `OBJECT` activity/ accommodation/ person/ travelplan | Deletes the given object type
 `edit -OBJECT` | `OBJECT` activity/ accommodation/ person/ travelplan | Edits the details of the given object type
 `goto PLAN_NAME` | `PLAN_NAME` nameOfTravelPlan/ wishlist | Navigate to the specific travel plan/ wishlist
+`find -OBJECT` | `OBJECT` activity/ accommodation/ person | Finds the given object type whose names contain any of the given keywords
```
Comment (jiaweiteo) [inline docs/UserGuide.md:66]: i think for all the find  command in the table, we should change to `COMMAND -OBJECT KEYWORD`, then the parameters add in `KEYWORD find object that contains this keyword`

Comment (jiaweiteo) [reply docs/UserGuide.md:66]: I did this for my delete command, think we shld do it for all commands on the table. Add the additional parameter for the commands that require it. Like `delete -object index`

**Review by jiaweiteo — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -58,6 +63,7 @@ Command | Parameters | Description
 `delete -OBJECT` | `OBJECT` activity/ accommodation/ person/ travelplan | Deletes the given object type
 `edit -OBJECT` | `OBJECT` activity/ accommodation/ person/ travelplan | Edits the details of the given object type
 `goto PLAN_NAME` | `PLAN_NAME` nameOfTravelPlan/ wishlist | Navigate to the specific travel plan/ wishlist
+`find -OBJECT KEYWORD` | `OBJECT` activity/ accommodation/ person<br/> `KEYWORD` keywords to search for  | Finds the given object type whose names contain any of the given keywords
```
Comment (underthehai) [inline docs/UserGuide.md:66]: find friend instead of person?

**Pull request merged**